README.md

```diff
@@ -1,289 +1,117 @@
-│ USA     ┆ Alabama  ┆ 2022-07-07 ┆ Homo sapiens ┆ 22A   │
-│ USA     ┆ Arizona  ┆ 2022-07-02 ┆ Homo sapiens ┆ 22B   │
-│ USA     ┆ Arizona  ┆ 2022-07-19 ┆ Homo sapiens ┆ 22B   │
-│ USA     ┆ Arizona  ┆ 2022-07-15 ┆ Homo sapiens ┆ 22B   │
-│ USA     ┆ Arizona  ┆ 2022-07-20 ┆ Homo sapiens ┆ 22B   │
-└─────────┴──────────┴────────────┴──────────────┴───────┘
+Cladetime knows where to find past versions of Nextstrain's SARS-CoV-2 sequence data and metadata files.
+These are [intermediate files produced by Nextstrain's daily workflow](https://docs.nextstrain.org/projects/ncov/en/latest/reference/remote_inputs.html#remote-inputs-open-files). Cladetime uses the full set of open Genbank data.
 
-# Pandas users can create a Pandas dataframe with sequence metadata
-
-In [8]: pandas = lf.collect().to_pandas()
-
-# Metadata from the pipeline that produced the above sequence_data
-In [9]: ct.ncov_metadata
-Out[9]:
-{'schema_version': 'v1',
- 'nextclade_version': 'nextclade 3.8.2',
- 'nextclade_dataset_name': 'SARS-CoV-2',
- 'nextclade_dataset_version': '2024-09-25--21-50-30Z',
- 'nextclade_tsv_sha256sum': 'fbe579554e925e4dfaf74cfb4e72b52c702e671f0f0374d896f1e30ae4fe5566',
- 'metadata_tsv_sha256sum': '5a4fd84a5cd3c4ead9cf730d4df10b8734898c6c3e0cae1c8c0acf432325d22c'}
- ```
-
- #### Work with point-in-time Nextstrain Sars-Cov-2 sequence metadata and clade assignments
-
- ```python
-In [10]: from cladetime import CladeTime
-
-In [11]: ct = CladeTime(sequence_as_of="2024-08-31", tree_as_of="2024-08-01")
-
-# URL for the corresponding Nextstrain Sars-Cov-2 sequence metadata as it existing on 2024-08-31
-In [12]: ct.url_sequence_metadata
-Out[12]: 'https://nextstrain-data.s3.amazonaws.com/files/ncov/open/metadata.tsv.zst?versionId=1SZMfjWxXjNy530F6L7MfyflUCbue.JD'
-
-# Metadata for the pipeline run that produced the above file
-In [13]: ct.ncov_metadata
-Out[13]: {'schema_version': 'v1',
- 'nextclade_version': 'nextclade 3.8.2',
- 'nextclade_dataset_name': 'SARS-CoV-2',
- 'nextclade_dataset_version': '2024-07-17--12-57-03Z',
- 'nextclade_tsv_sha256sum': 'fd30f0b258f73fdcf5acefe77937ebe7d88862093bb4aaf3a7e935650ccea060',
- 'metadata_tsv_sha256sum': '898451d9750128b4f90253d91cef0092e51965e879536e80aa6598de0fd4af29'}
+```python
+>>> from cladetime import CladeTime
+
+# Create a CladeTime object for any date after May 2023
+# (for the most recent data, remove the `sequence_as_of` parameter)
+>>> ct = CladeTime(sequence_as_of="2024-10-15")
+
+# Based on the sequence_as_of date (above), Cladetime provides URLs to the corresponding
+# SARS-CoV-2 past sequence data and metadata files
+>>> ct.url_sequence
+'https://nextstrain-data.s3.amazonaws.com/files/ncov/open/sequences.fasta.zst?versionId=8Zszokay3LRP5Zec_cviQ8oXkx8cJlwq'
+>>> ct.url_sequence_metadata
+'https://nextstrain-data.s3.amazonaws.com/files/ncov/open/metadata.tsv.zst?versionId=U4aIlh5HI1XuDLPW7q9WTZad6gXwARqT'
 ```
 
+### Interacting with sequence metadata
 
-## Docker Setup
```

Comment:
Opened an issue to [move the setup instructions to a new CONTRIBUTING.md file](https://github.com/reichlab/cladetime/issues/40).

At this point, the dev setup distracts from the main focus of the page, which is how to get started.